README.md

```diff
@@ -22,22 +28,38 @@ import vendor_data as vd
 
 Check out `tutorial.md` to see an example of the different functionalities of the script!
 
+### Running the script
 You can also run the script directly with the following code in a console:
 
 ```sh
-python vendor_data.py <example.json>
+python vendor_data.py <"-a/--add or -m/--modify">
 ```
 
 Or in Jupyter notebook with:
 
 ```python
-%run vendor_data.py <example.json>
+%run vendor_data.py <"-a/--add or -m/--modify">

```

Comment:
In both cases, because you have two mutually-exclusive arguments, I would make to chunks to show. That way, a user can copy your code. So: one line showing how to call your script to add an entry, and another line showing how to call your script to modify an entry.